lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -11,6 +11,7 @@
 
 import numpy as np
 
+from matplotlib import _api
```

Comment:
This belongs in the next block, not here (and actually `contextlib` should be in the first block, but that's unrelated here). And `_api` can go directly into line 18.